### PR TITLE
Dev 25 - Word Cloud functionality

### PR DESCRIPTION
**Pull request by KovDimaY**
Look at the issue #46

### CODE REVIEW

**Comment by KovDimaY** [comment]
I am waiting for reviews for 1 hour and then merge it if there is no comments.

File: src/components/charts/other.js [hunk 1]
```diff
@@ -350,6 +366,39 @@ export default class Other extends Component {
     this.setState({ configurations });
   }
 
+  onWordcloudInputChange(event) {
+    const { configurations } = this.state;
+    const newValue = event.target.value === ''
+      ? 'Enter here your text to plot its words set on the chart...'
```
Comment (KovDimaY) [inline src/components/charts/other.js]: The chart does not work with an empty data...

File: src/constants/other/default-options-other.js [hunk 2]
```diff
@@ -378,107 +378,16 @@ export const pyramid = {
...
-          {"name":"neque","weight":1},
-          {"name":"dui","weight":1},
-          {"name":"maximus","weight":1},
-          {"name":"id","weight":1},
-          {"name":"Nullam","weight":1},
-          {"name":"vel","weight":1},
-          {"name":"lorem","weight":1},
-          {"name":"","weight":1}
-        ]
+        data: [{ name: 'init', weight: 1 }]
```
Comment (KovDimaY) [inline src/constants/other/default-options-other.js]: The chart does not work with an empty data...

File: src/components/charts/other.js (hunk 1 above)
Comment (NikolayYakovenko) [reply src/components/charts/other.js]: In my opinion, if you remove all text on focus event, the usage of this textarea will be more comfortable.
Especially, when user copy and paste text from another place.

Comment (KovDimaY) [reply src/components/charts/other.js]: I agree! Good idea! Thank you!

File: src/constants/other/default-options-other.js [hunk 3]
```diff
@@ -378,107 +378,16 @@ export const pyramid = {
...
     }],
     title: {
-        text: 'Wordcloud of Lorem Ipsum'
+        text: ''
+    },
+    legend: {
+        enabled: false
+    },
+    tooltip: {
+      enabled: true,
```
Comment (KovDimaY) [inline src/constants/other/default-options-other.js]: This is wrong, it should be false

**Pull request merged**